cmake/Modules/Pod2Man.cmake

```diff
@@ -58,7 +58,7 @@ MACRO(POD2MAN PODFILE MANFILE SECTION)
 
 	INSTALL(
 		FILES ${CMAKE_CURRENT_BINARY_DIR}/${MANFILE}.${SECTION}.gz
-		DESTINATION share/man/man${SECTION}
+		DESTINATION ${MANPAGE_DEST_PREFIX}/man${SECTION}
     	)
 ENDMACRO(POD2MAN PODFILE MANFILE SECTION)
```

Comment:
I believe we need to reflect macro changes here as well

Reply:
Agreed. This has been updated.